File: backend/src/main/java/com/revnu/backend/features/staff/dto/SalaryResponse.java
package com.revnu.backend.features.staff.dto;

import java.math.BigDecimal;
import java.time.LocalDate;
import java.time.LocalDateTime;
import java.util.UUID;

import com.revnu.backend.features.staff.model.SalaryStatus;

public record SalaryResponse(
        UUID id,
        UUID staffId,
        String staffName,
        BigDecimal amount,
        LocalDate paymentDate,
        SalaryStatus status,
        LocalDateTime createdAt
        ) {

}

File: mobile/app/src/main/java/com/revnu/mobile/features/staff/model/SalaryResponse.kt
package com.revnu.mobile.features.staff.model

import com.google.gson.annotations.SerializedName
import java.io.Serializable
import java.math.BigDecimal



data class SalaryResponse(
    @SerializedName("id") val id: String,
    @SerializedName("staffId") val staffId: String,
    @SerializedName("staffName") val staffName: String,
    @SerializedName("amount") val amount: Double,
    @SerializedName("paymentDate") val paymentDate: String,
    @SerializedName("status") val status: String,
    @SerializedName("createdAt") val createdAt: String
)